standup/api/views.py

```diff
@@ -190,3 +190,50 @@ def post(self, request):
         status.save()
 
         return HttpResponseJSON({'id': status.id, 'content': content})
+
+
+class StatusDelete(AuthenticatedAPIView):
+    def delete(self, request, pk):
+        token = request.auth_token
+
+        # FIXME: Authorize this operation.
+
+        username = request.json_body.get('user')
+
+        if not username:
+            return HttpResponseBadRequest('Missing required fields.')
+
+        status = Status.objects.filter(id=pk).first()
+        if not status:
+            return HttpResponseBadRequest('Status does not exist.')
+
+        if status.user.username != username:
+            return HttpResponseForbidden('You cannot delete this status.')
+
+        status_id = status.id
+        status.delete()
+
+        return HttpResponseJSON({'id': status_id})
+
+
+class UpdateUser(AuthenticatedAPIView):
+    def post(self, request, username):
+        token = request.auth_token
+
+        # FIXME: Authorize this operation.
+
+        user = StandupUser.objects.filter(user__username=username).first()
```

Comment:
I think this should search based on `slug`. Currently the Profile page uses the `slug` field for "IRC Handle".